juno/assetlist.json

```diff
@@ -1242,6 +1242,28 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/spacer.png"
       }            
     },
+    {
+      "description":"Light: LumenX community DAO treasury token",
+      "type_asset": "cw20",
+      "address": "juno1dpany8c0lj526lsa02sldv7shzvnw5dt5ues72rk35hd69rrydxqeraz8l"
+          "denom_units":[
+             {
+                "denom":"cw20:juno1dpany8c0lj526lsa02sldv7shzvnw5dt5ues72rk35hd69rrydxqeraz8l",
+                "exponent":0
+             },
+             {
+                "denom":"light",
+                "exponent":9
+             }
+          ],
+       "base":"nlight",
```

Comment:
Thanks for making the changes, although this 'base' property must exactly refer to a defined denom_unit, and therefore should also be "cw20:juno1dpany8c0lj526lsa02sldv7shzvnw5dt5ues72rk35hd69rrydxqeraz8l"